kk/tests/test_hearing.py

```diff
@@ -95,6 +95,33 @@ def test_get_next_closing_hearings(self):
         assert len(data) == 1
         assert data[0]['abstract'] == future_hearing_2.abstract
 
+    def test_list_hearings_check_references_not_included(self):
+        # DynamicFieldsMixin (https://gist.github.com/dbrgn/4e6fc1fe5922598592d6)
+        pytest.xfail("Test fields requested dynamicaly for a list")
+
+    def test_8_get_detail_check_properties(self, default_hearing):
+        response = self.client.get(self.get_hearing_detail_url(default_hearing.id))
+        assert response.status_code is 200
+
+        data = self.get_data_from_response(response)
+
+        assert 'abstract' in data
```

Comment:
You _could_ just do something like

```python
assert all(k in data for k in HearingSerializer._meta.fields)
```

to avoid having to re-list the fields here...